mutiny-wasm/src/indexed_db.rs

```diff
@@ -255,36 +255,41 @@ impl IndexedDbStorage {
 
         match kv.key.as_str() {
             NODES_KEY => {
-                let obj = vss.get_object(&kv.key).await?;
                 // we can get version from node storage, so we should compare
                 match current.get_data::<NodeStorage>(&kv.key)? {
                     Some(versioned) => {
-                        if versioned.version <= kv.version
-                            && serde_json::from_value::<NodeStorage>(obj.value.clone()).is_ok()
-                        {
-                            return Ok(Some((kv.key, obj.value)));
+                        if versioned.version < kv.version {
```

Comment:
can `versioned.version` be renamed? it's very confusing. it sounds like it should be called `local.version`.